config/jobs/preflight/preflight-presubmits.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  jetstack/preflight:
+
+  - name: preflight-unit-tests
+    cluster: gke
+    agent: kubernetes
+    decorate: true
+    decoration_config:
+      ssh_key_secrets:
+      - jetstack-pulling-bot
+    clone_uri: "git@github.com:jetstack/preflight.git"
+    always_run: true
+    trigger: "(?m)^/test"
+    rerun_command: "/test"
```

Comment:
I think you can remove the `trigger` line, and then change `rerun_command` to `/test preflight-unit-tests`

Reply:
AFAIK `trigger` and `rerun_command` need to exist (I got an error before because `trigger` wasn't defined).

Reply:
> {"component":"checkconfig","error":"Either both of job.Trigger and job.RerunCommand must be set, wasnt the case for job \"preflight-unit-tests\"","file":"prow/cmd/checkconfig/main.go:180","func":"main.main","level":"fatal","msg":"Error loading Prow config.","time":"2019-11-12T18:08:46Z"}

exactly, `trigger` is also needed.